extensions/import/src/wizard/dacfxExportWizard.ts

```diff
@@ -0,0 +1,127 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+'use strict';
+import * as vscode from 'vscode';
+import * as nls from 'vscode-nls';
+import * as sqlops from 'sqlops';
+import { ExportConfigPage } from './pages/exportConfigPage';
+import { DacFxDataModel } from './api/models';
+import { ExportSummaryPage } from './pages/exportSummaryPage';
+import { DacFxPage } from './api/dacFxPage';
+import { DacFxWizard } from './dacfxWizard';
+
+const localize = nls.loadMessageBundle();
+
+export class DacFxExportWizard extends DacFxWizard {
+	private wizard: sqlops.window.modelviewdialog.Wizard;
+	private connection: sqlops.connection.Connection;
+	private exportConfigPage: ExportConfigPage;
+	private summaryPage: ExportSummaryPage;
+	private model: DacFxDataModel;
+
+	constructor() {
+		super();
+	}
+
+	public async start(p: any, ...args: any[]) {
+		this.model = <DacFxDataModel>{};
+		let pages: Map<number, DacFxPage> = new Map<number, DacFxPage>();
+
+		let profile = p ? <sqlops.IConnectionProfile>p.connectionProfile : null;
```

Comment:
we are trying to use `undefined` instead of `null` through the code base.  Could you please switch all `null` references to `undefined`?